Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -111,35 +112,35 @@ public BlockResult ExecuteNextBlock(CancellationToken cancellationToken)
             {
                 result.Error = ex.ConsensusError;
             }
+
             return result;
         }
 
         public void AcceptBlock(ContextInformation context)
         {
             using (this.watch.Start(o => this.Validator.PerformanceCounter.AddBlockProcessingTime(o)))
             {
-                // check that the current block has not been reorged
-                // catching a reorg at this point will not require a rewind
+                // Check that the current block has not been reorged.
+                // Catching a reorg at this point will not require a rewind.
```

Comment:
This is a condition I have added, I don't think core or X do such a thing. but also we use consensus a bit different.